GOVERNANCE.md

```diff
@@ -0,0 +1,179 @@
+# Overview
+
```

Comment:
Thank you for working on this. IMHO it is too much text, though. Since a lot of text seems to be too vague or not really fitting to Nmstate, I would prefer to mention only the parts that are relevant for now.

Reply:
I have not found points that are not relevant.
Please point them out and try to help in suggesting an alternative.